client-js/app/scripts/manager.ts

```diff
@@ -53,7 +61,7 @@ export interface ManagerApi {
   serverDisconnected: Observable<ServerDisconnectMessage>;
 }
 
-export class ManagerServiceJsonRpc extends rpc.RpcService implements ManagerApi {
+export class ManagerApiImpl extends rpc.RpcService implements ManagerApi {
```

Comment:
That's because code that uses this is written in terms of the interface, so it gets the nice name. I realize that I named this inconsistently from other classes which use `Service` as a suffix instead of `Impl`; I like `Impl` better but didn't want to change code to match that is otherwise unrelated to this PR.